bundled/tool/check_consent.py

```diff
@@ -0,0 +1,29 @@
+from kedro_telemetry.plugin import _check_for_telemetry_consent
+from kedro_telemetry.plugin import (
+    _get_project_properties,
+    _get_or_create_uuid,
+)
+import sys
+import json
```

Comment:
```suggestion
from common import update_sys_path
from pathlib import Path
import os
import sys
import json

update_sys_path(
    os.fspath(Path(__file__).parent.parent / "libs"),
    os.getenv("LS_IMPORT_STRATEGY", "useBundled"),
)
```

I tired the above changes and it worked for me.  Let me know if its a right way to do it.